dev/serializing_util.h

```diff
@@ -422,11 +422,15 @@ namespace sqlite_orm {
                     }
                 }
             } else {
-                constexpr bool hasUnindexedOption =
-                    mpl::invoke_t<check_if_has_type<unindexed_t>, constraints_tuple>::value;
-                if SQLITE_ORM_CONSTEXPR_IF(hasUnindexedOption) {
-                    ss << " UNINDEXED";
-                }
+                iterate_tuple(column.constraints, [&ss, &context](auto& constraint) {
```

Comment:
It's easier to get a filtered index sequence:
```c++
                using fts5_index_sequence = filter_tuple_sequence_t<
                    constraints_tuple,
                    mpl::disjunction<check_if_is_type<unindexed_t>, check_if_is_template<prefix_t>>::template fn>;
                iterate_tuple(column.constraints, fts5_index_sequence{}, [&ss, &context](auto& constraint) {
                    ss << ' ' << serialize(constraint, context);
```

Reply:
@fnc12 II have an even better suggestion. The question is, why do we need to differentiate between constraint types here at all?
You can never add regular column constraints to FTS5 columns and vice versa. In any case, it is better to prevent this when creating the column, not when serializing it.

So, this implementation is much simpler:
```c++
template<class... Op, class Ctx>
std::ostream& operator<<(std::ostream& ss,
                         std::tuple<const streaming<stream_as::column_constraints>&,
                                    const column_constraints<Op...>&,
                                    const bool&,
                                    Ctx> tpl) {
    const auto& column = std::get<1>(tpl);
    const bool& isNotNull = std::get<2>(tpl);
    auto& context = std::get<3>(tpl);

    using constraints_tuple = decltype(column.constraints);
    iterate_tuple(column.constraints, [&ss, &context](auto& constraint) {
        ss << ' ' << serialize(constraint, context);
    });
    // add implicit null constraint
    if(!context.fts5_columns) {
        constexpr bool hasExplicitNullableConstraint =
            mpl::invoke_t<mpl::disjunction<check_if_has_type<null_t>, check_if_has_type<not_null_t>>,
                          constraints_tuple>::value;
        if SQLITE_ORM_CONSTEXPR_IF(!hasExplicitNullableConstraint) {
            if(isNotNull) {
                ss << " NOT NULL";
            } else {
                ss << " NULL";
            }
        }
    }

    return ss;
}
```

Reply:
@trueqbit the case is that SQLite FTS docs say that one can add regular column constraints to FTS columns but those constraints will be ignored by SQLite engine. That's why I decided to remove them from serialization. But let's try your idea

Reply:
Interesting - I haven't read all the details... I would say that in this case it would even argue for not checking the restrictions on FTS5 columns.